Orange/widgets/data/owimpute.py

```diff
@@ -348,6 +376,20 @@ def imputed_vars_for_column(self, column):
         else:
             assert False
 
+    def _handle_commit(func):
```

Comment:
The name is a bit vague. When and why is it needed? The only reference point available to a future explorer is the commit hash which ties things together. I tend to dislike this.